assignments/prototypes.js

```diff
@@ -1,110 +1,161 @@
-    team: 'Mage Guild',
+    name: "Bruce",
+    team: "Mage Guild",
     weapons: [
-      'Staff of Shamalama',
+        "Staff of Shamalama",
     ],
-    language: 'Common Tongue',
-  });
+    language: "Common Tongue",
+});
 
-  const swordsman = new Humanoid({
+const swordsman = new Humanoid({
     createdAt: new Date(),
     dimensions: {
-      length: 2,
-      width: 2,
-      height: 2,
+        length: 2,
+        width: 2,
+        height: 2,
     },
     healthPoints: 15,
-    name: 'Sir Mustachio',
-    team: 'The Round Table',
+    name: "Sir Mustachio",
+    team: "The Round Table",
     weapons: [
-      'Giant Sword',
-      'Shield',
+        "Giant Sword",
+        "Shield",
     ],
-    language: 'Common Tongue',
-  });
+    language: "Common Tongue",
+});
 
-  const archer = new Humanoid({
+const archer = new Humanoid({
     createdAt: new Date(),
     dimensions: {
-      length: 1,
-      width: 2,
-      height: 4,
+        length: 1,
+        width: 2,
+        height: 4,
     },
     healthPoints: 10,
-    name: 'Lilith',
-    team: 'Forest Kingdom',
+    name: "Lilith",
+    team: "Forest Kingdom",
     weapons: [
-      'Bow',
-      'Dagger',
+        "Bow",
+        "Dagger",
     ],
-    language: 'Elvish',
-  });
+    language: "Elvish",
+});
 
-  console.log(mage.createdAt); // Today's date
-  console.log(archer.dimensions); // { length: 1, width: 2, height: 4 }
-  console.log(swordsman.healthPoints); // 15
-  console.log(mage.name); // Bruce
-  console.log(swordsman.team); // The Round Table
-  console.log(mage.weapons); // Staff of Shamalama
-  console.log(archer.language); // Elvish
-  console.log(archer.greet()); // Lilith offers a greeting in Elvish.
-  console.log(mage.takeDamage()); // Bruce took damage.
-  console.log(swordsman.destroy()); // Sir Mustachio was removed from the game.
-*/
+console.log(mage.createdAt); // Today"s date
+console.log(archer.dimensions); // { length: 1, width: 2, height: 4 }
+console.log(swordsman.healthPoints); // 15
+console.log(mage.name); // Bruce
+console.log(swordsman.team); // The Round Table
+console.log(mage.weapons); // Staff of Shamalama
+console.log(archer.language); // Elvish
+console.log(archer.greet()); // Lilith offers a greeting in Elvish.
+console.log(mage.takeDamage()); // Bruce took damage.
+console.log(swordsman.destroy()); // Sir Mustachio was removed from the game.
 
-  // Stretch task: 
-  // * Create Villain and Hero constructor functions that inherit from the Humanoid constructor function.  
-  // * Give the Hero and Villains different methods that could be used to remove health points from objects which could result in destruction if health gets to 0 or drops below 0;
-  // * Create two new objects, one a villain and one a hero and fight it out with methods!
\ No newline at end of file
+
+// Stretch task:
+// * Create Villain and Hero constructor functions that inherit from the Humanoid constructor function.
+function Villain(spec) {
```

Comment:
Great job on getting the stretch problem going! If you ever find yourself with time on the side after completion you might want to try and really push yourself. You have one of the best understandings of JS that I've seen in the cohort, and I think that you could do some amazing things if you hit this hard..